packages/auth/src/authClients/nhost.ts

```diff
@@ -1,10 +1,10 @@
+import { AuthClient } from './'
+
 import type { Session, NhostClient, User } from 'nhost-js-sdk'
 
```

Comment:
This is Prettier's formatting, which ESLint doesn't like:
<img width="593" alt="Screen Shot 2021-05-14 at 12 22 48 PM" src="https://user-images.githubusercontent.com/2951/118319063-19334900-b4af-11eb-9d3f-8c00e2884930.png">